include/seqan3/std/iterator

```diff
@@ -301,6 +301,11 @@ SEQAN3_CONCEPT ContiguousIterator = RandomAccessIterator<i> &&
                                     ::ranges::ContiguousIterator<i>;
 //!\endcond
 
+/*!\typedef std::iter_reference_t
```

Comment:
It may fit better next to line 106?  (`iter_difference_t`)
But not so important.